SPECS/node-problem-detector/node-problem-detector.spec

```diff
@@ -37,11 +37,10 @@ Source1:        %{name}-%{version}-vendor.tar.gz
 #           -cf %%{name}-%%{version}-test-vendor.tar.gz vendor
 #
 Source2:        %{name}-%{version}-test-vendor.tar.gz
-Patch0:         0001-remove-arch-specific-logic-from-makefile.patch
-Patch1:         0001-add-Mariner-and-Azure-Linux-OS-Versions.patch
-Patch2:         CVE-2024-24786.patch
-Patch3:         CVE-2024-45338.patch
-Patch4:         CVE-2023-45288.patch
+Patch1:         0001-remove-arch-specific-logic-from-makefile.patch
+Patch2:         CVE-2024-45338.patch
+Patch3:         CVE-2025-22868.patch
+Patch4:         CVE-2025-22869.patch
```

Comment:
Please remove the patch files which are no longer needed.

Reply:
The source files for the patches are not yet removed though. They are otherwise left hanging in the spec folder without actually being necessary for the package.